tests/UnifiedSpecTests/command-monitoring/unacknowledgedBulkWrite.json

```diff
@@ -70,17 +70,7 @@
           "object": "collection",
           "arguments": {
             "filter": {}
-          },
-          "expectResult": [
```

Comment:
This creates a difference with the [spec file](https://github.com/mongodb/mongo-php-library/blob/7d3b00947f5f4405b31f7cb2e08315c8ac69e950/tests/UnifiedSpecTests/command-monitoring/unacknowledgedBulkWrite.json). How do you intend to return to the spec file ?

Reply:
The corresponding pull request is already merged: https://github.com/mongodb/specifications/pull/1604